src/pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -57,27 +57,20 @@ def __init__(
         self.device = device
         self.scaler = scaler
 
-    def pre_backward(self, model: "pl.LightningModule", closure_loss: Tensor) -> Tensor:
-        if self.scaler is not None:
-            closure_loss = self.scaler.scale(closure_loss)
-        return super().pre_backward(model, closure_loss)
-
-    def _run_backward(self, tensor: Tensor, model: Optional[Module], *args: Any, **kwargs: Any) -> None:
+    def pre_backward(self, tensor: Tensor, module: "pl.LightningModule") -> Tensor:  # type: ignore[override]
         if self.scaler is not None:
             tensor = self.scaler.scale(tensor)
-        super()._run_backward(tensor, model, *args, **kwargs)
+        return super().pre_backward(tensor, module)
 
-    def optimizer_step(
-        self,
-        model: Optional[Union["pl.LightningModule", Module]],
-        optimizer: Optimizer,
-        optimizer_idx: int,
-        closure: Callable[[], Any],
-        **kwargs: Any,
-    ) -> Any:
+    def optimizer_step(self, optimizer: Steppable, **kwargs: Any) -> Any:
+        optimizer_idx = kwargs.pop("optimizer_idx")
```

Comment:
Why don't you expose them in the method signature ?

Reply:
In Lite, there is no optimizer_idx